tests/renderNothing.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable max-nested-callbacks, react/prefer-stateless-function, class-methods-use-this */
```

Comment:
Whats the reason for needing `react/prefer-stateless-function, class-methods-use-this` ?